doc/source/conf.py

```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
```

Comment:
@MatteoRobbiati please install pre-commit locally to avoid these pushes.

Reply:
Yes, sorry @scarrazza . 
I have just done it with the help of Andrea. I downloaded it but now it is installed.